projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -1065,12 +1065,16 @@ public static function notice( $message, $status = 'info', $classes = '' ) {
 	 * @param string $slug Slug of the block.
 	 */
 	public static function set_availability_for_plan( $slug ) {
-		$is_available   = true;
-		$plan           = '';
-		$slug           = self::remove_extension_prefix( $slug );
-		$features_data  = array();
-		$is_simple_site = defined( 'IS_WPCOM' ) && IS_WPCOM;
-		$is_atomic_site = ( new Host() )->is_woa_site();
+		$is_available    = true;
+		$plan            = '';
+		$slug            = self::remove_extension_prefix( $slug );
+		$features_data   = array();
+		$is_simple_site  = defined( 'IS_WPCOM' ) && IS_WPCOM;
+		$is_atomic_site  = ( new Host() )->is_woa_site();
+		$current_blog_id = get_current_blog_id();
```

Comment:
Since we're only using this inside the wpcom / atomic check, should we move the variable declaration there as well?

Reply:
Sure thing, updated.